lawnchair/res/values/themes.xml

```diff
@@ -31,4 +31,14 @@
         <item name="android:enforceNavigationBarContrast">false</item>
         <item name="windowNoTitle">true</item>
     </style>
+
+    <style name="Theme.AppCompat.Translucent.NoTitleBar" parent="Theme.AppCompat.NoActionBar">
+        <item name="android:windowBackground">@android:color/transparent</item>
+        <item name="android:colorBackgroundCacheHint">@null</item>
+        <item name="android:windowIsTranslucent">true</item>
+        <!-- Note that we use the base animation style here (that is no
+                 animations) because we really have no idea how this kind of
+                 activity will be used. -->
+        <item name="android:windowAnimationStyle">@android:style/Animation</item>
+    </style>
```

Comment:
```xml
<!-- Theme for translucent activities (on API level 10 and lower). That is, windows
    that allow you to see through them to the windows behind.  This sets up the translucent
    flag and appropriate animations for your windows.  -->
<style name="Theme.Translucent">
    <item name="windowBackground">@color/transparent</item><item name="colorBackgroundCacheHint">@null</item><item name="windowIsTranslucent">true</item>< !-- Note that we use the base animation style here (that is no animations) because we really have no idea how this kind of activity will be used. --><item name="windowAnimationStyle">@style/Animation</item>
</style>

<!-- Variant of {@link #Theme_Translucent} with no title bar -->
<style name="Theme.Translucent.NoTitleBar">
    <item name="windowNoTitle">true</item>
    <item name="windowContentOverlay">@null</item>
</style>
```